From f668c59c1aeaf5a4ca995faacb415c0c0493d734 Mon Sep 17 00:00:00 2001
From: Silvio Traversaro <silvio@traversaro.it>
Date: Fri, 1 Aug 2025 11:26:27 +0200
Subject: [PATCH] Fix compilation of executorDataVector.cpp in Linux on arm64

---
 pxr/exec/vdf/executorDataVector.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/pxr/exec/vdf/executorDataVector.cpp b/pxr/exec/vdf/executorDataVector.cpp
index 12f9e517e0..96dacca064 100644
--- a/pxr/exec/vdf/executorDataVector.cpp
+++ b/pxr/exec/vdf/executorDataVector.cpp
@@ -24,7 +24,7 @@ PXR_NAMESPACE_OPEN_SCOPE
 static inline void
 Vdf_ExecutorDataVector_ValgrindMakeDefined(void *ptr, size_t size)
 {
-#if defined(ARCH_OS_LINUX)
+#if defined(ARCH_OS_LINUX) && defined(ARCH_CPU_INTEL)
     // Pass a result and a pointer to some arguments via registers
     volatile unsigned long long int result;
     volatile unsigned long long int args[6] = {
